packages/components/psammead-podcast-promo/src/components/card-image-wrapper.jsx

```diff
@@ -7,18 +7,13 @@ import {
 } from '@bbc/gel-foundations/breakpoints';
 
 const CardImageWrapper = styled.div`
-  /* 88px */
```

Comment:
_"A code comment is a lie waiting to be told"_, @jroebu14 2021 (I know you didn't come up with it but hey it's 2021 - proper attribution is a myth).

Reply:
haha I found the proper attribution which I seemed to have paraphrased

> a comment is a lie waiting to happen
Josh Susser

here's another
> A comment is the code’s way of asking to be more clear
Kent Beck

another good one
> Clear code with no comments is preferable to messy code with clear comments.

The above example is definitely a lie waiting to happen 😆 

Comments are an interesting debate though. Personally I would rarely write comments. I'd think about how the code can be clearer or how a unit test could explain something better since it's harder to lie with unit tests!